x-pack/plugins/lens/public/pie_visualization/suggestions.ts

```diff
@@ -45,7 +67,11 @@ export function suggestions({
 
   const [groups, metrics] = partition(table.columns, (col) => col.operation.isBucketed);
 
-  if (metrics.length > 1 || groups.length > Math.max(MAX_PIE_BUCKETS, MAX_TREEMAP_BUCKETS)) {
+  if (
+    metrics.length > 1 ||
+    groups.length >
+      Math.max(...Object.values(PartitionChartsMeta).map(({ maxBuckets }) => maxBuckets))
```

Comment:
nit: could you extract it to the const that would have some descriptive name? I find it hard to understand what this check does